### ISSUE TITLE
Performance test - Unique index or primary key violation 

### ISSUE THREAD

**Issue by rr83019** [librecaptcha/lc-core#44]
This error occurs on `/captcha` endpoint 
```
<Response [500]>
<!DOCTYPE html>
<html>
<head><title>500 Internal Server Error</title></head>
<body><h1>500 Internal Server Error</h1>
<p>Error processing request: Unique index or primary key violation: &quot;PUBLIC.PRIMARY_KEY_4 ON PUBLIC.MAPID(UUID) VALUES 36797&quot;; SQL statement:
INSERT INTO mapId(uuid, token) VALUES (?, ?) [23505-200]</p>
</body></html>

**Comment by rr83019** [comment]
Stack trace: 

```
[info] org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: Referential integrity constraint violation: "CONSTRAINT_45: PUBLIC.MAPID FOREIGN KEY(TOKEN) REFERENCES PUBLIC.CHALLENGE(TOKEN) ('9668bb65-0bf0-455a-866d-e8c7ebb272a4')"; SQL statement:
[info] INSERT INTO mapId(uuid, token) VALUES (?, ?) [23506-200]
[info] 	at org.h2.message.DbException.getJdbcSQLException(DbException.java:459)
[info] 	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
[info] 	at org.h2.message.DbException.get(DbException.java:205)
[info] 	at org.h2.message.DbException.get(DbException.java:181)
[info] 	at org.h2.constraint.ConstraintReferential.checkRowOwnTable(ConstraintReferential.java:319)
[info] 	at org.h2.constraint.ConstraintReferential.checkRow(ConstraintReferential.java:261)
[info] 	at org.h2.table.Table.fireConstraints(Table.java:1057)
[info] 	at org.h2.table.Table.fireAfterRow(Table.java:1075)
[info] 	at org.h2.command.dml.Insert.insertRows(Insert.java:211)
[info] 	at org.h2.command.dml.Insert.update(Insert.java:151)
[info] 	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
[info] 	at org.h2.command.Command.executeUpdate(Command.java:251)
[info] 	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
[info] 	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
[info] 	at lc.Captcha.getUUID(Main.scala:118)
[info] 	at lc.Captcha.getChallenge(Main.scala:109)
[info] 	at lc.Server.$anonfun$new$1(Server.scala:75)
[info] 	at lc.HTTPServer.serve(Unknown Source)
[info] 	at lc.HTTPServer.handleMethod(Unknown Source)
[info] 	at lc.HTTPServer.handleTransaction(Unknown Source)
[info] 	at lc.HTTPServer.handleConnection(Unknown Source)
[info] 	at lc.HTTPServer$SocketHandlerThread$1.run(Unknown Source)
[info] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[info] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[info] 	at java.base/java.lang.Thread.run(Thread.java:834)

**Comment by hrj** [comment]
Closed via 62b3a098bdbe71ecd33194677e58801195b73682
